docs/index.rst

```diff
@@ -154,3 +199,7 @@ Indices and tables
 
 * :ref:`genindex`
 * :ref:`modindex`
+
+
+.. _docker: https://docs.docker.com/installation/#installation
```

Comment:
I like that way of putting all the references to the bottom of the file, maybe we should add some note to the docs (in the contributing?) about that (in another PR ;)